stripe-attestation/src/main/java/com/stripe/attestation/IntegrityStandardRequestManager.kt

```diff
@@ -49,9 +49,13 @@ class IntegrityStandardRequestManager(
 
         finishedTask.toResult()
             .onSuccess { integrityTokenProvider = it }
-            .onFailure { error -> logError("Integrity: Failed to prepare integrity token", error) }
             .getOrThrow()
     }
+        .map {}
+        .recoverCatching {
+            logError("Integrity - Failed to prepare integrity token", it)
+            throw AttestationError.fromException(it)
+        }
```

Comment:
this ensures any exceptions executing on integrity are wrapped on an `AttestationError` instance. Note exceptions still get wrapped on a `Result`.